lgsm/functions/core_functions.sh

```diff
@@ -7,7 +7,7 @@
 
 functionselfname="$(basename "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-modulesversion="v20.3.2"
```

Comment:
Versions should not be increased.
Please us for this the `clear-functions` of the dev commands for the tests